template.md

```diff
@@ -1,5 +1,5 @@
 
-<img src="../images/Ecma_RVB-003.jpg" align="right" height="70" alt="" />
+<img src="../images/Ecma_RVB-003.jpg" align="right" height="70" alt="Ecma logo" />
```

Comment:
might as well update the may agenda also

Reply:
✅